client/rest-high-level/src/main/java/org/elasticsearch/client/tasks/TaskGroup.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.client.tasks;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * Client side counterpart of server side version.
+ *
+ * {@link org.elasticsearch.action.admin.cluster.node.tasks.list.TaskGroup}
+ */
+public class TaskGroup {
```

Comment:
Im not sure we need any of this here. This is part of the cat api stuff mentioned above.

Reply:
https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/java-rest-high-cluster-cancel-tasks.html

they are part of the API, that's why I out them there.
Do they need to be removed?